scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -117,15 +120,30 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
         <!-- show transcript info only if it's canonical, disease-associated or primary (print max 5 primary transcripts) -->
       {% if transcript.refseq_identifiers and n_primary_txs.count <= 5 or transcript.is_canonical or transcript.is_disease_associated or transcript.is_primary %}
         <li>{{transcript.transcript_id}}, RefSeq:[{{ transcript.refseq_identifiers|join(", ") or "-" }}], {{ (transcript.coding_sequence_name or '')|truncate(20, True) }},  {{ (transcript.protein_sequence_name or '')|url_decode }}
+
+          <!-- Capture HGNC transcript primary transcript from canonical or primary transcript id or refseq id-->
+          {% if hgnc_primary.refseq is none and transcript.is_canonical or transcript.is_primary %}
+            {% if hgnc_primary.refseq is none and transcript.is_canonical and transcript.transcript_id.startswith('NM_') %}
```

Comment:
Here too: while often the same, it seems there is no guarantee `is_canonical`will find the right transcript

Starts with `NM_` is a sufficient but not necessary criteria: there may still be some `XM_` in HGNC.

Dunno, it is clearly better, but still not crystal clear to me. Unless you have checked very carefully, I feel we should not just guess on correcting this one? We will have to answer those "but my transcript is not in hg37" anyway, and I'd rather be sure the remainder are right! 😊

Reply:
It's not crystal clear to me either. I've tested it on several cases on stage and it looked fine but I can't be sure that it will always capture the right one. Because I'm not sure of the whole thing that is. I don't know, we could close this one and keep it as it is!

Reply:
Would there be any issue with just dropping the `and transcript.is_canonical` and leaving it at starts with NM_ or XM_, then matching to the actual primary id?

Reply:
I don't know about the result. Let's leave it out of the next release and let's test your suggestion with calm